tests/flakyfunctional/cylc-take-checkpoints/00-basic/flow.cylc

```diff
@@ -0,0 +1,25 @@
+[scheduler]
```

Comment:
What's this file doing here? Rebase error?

Reply:
Yep - I see - and Have removed.